mqtt/mqtt-broker/Cargo.toml

```diff
@@ -9,7 +9,7 @@ edition = "2018"
 [dependencies]
 async-trait = "0.1"
 bincode = "1.2"
-bytes = "0.5"
+bytes = "0.5.4"
```

Comment:
any specific reason for this change? other dependencies added without exact build number

Reply:
just a merge change. We both independently added it and your change went in first. I'll take off the 4